task/testmap.py

```diff
@@ -140,16 +140,15 @@
             'fedora-30/edge',
             'fedora-31/firefox',
             'rhel-7-8/firefox',
-            'rhel-8-1/chrome',
+            'rhel-8-2/chrome',
```

Comment:
We would lose RHEL 8.1 tests on master with that. I would recommend to keep 8.1 until 8.2 gets released. Instead, fedora-30 is rather over-represented here, how about moving one f-30 test to f31, one to rhel-8-2, and keep the third as f-30?